### DISCUSSION TITLE
cant resolve module 'fs' or 'net'

### DISCUSSION THREAD

**Discussion by aaronmgdr** [celo-org/celo-monorepo#9578] [Q&A] [answered]
if you get an error like cant resolve module 'fs' and are using ContractKit on Browser try updating your web config like the gist linked

https://gist.github.com/nestorbonilla/505f1cfbce73f868c6dc028833bdbd2b

**Comment by aaronmgdr** [comment, marked as answer]
if you are using create-react-app 

https://github.com/facebook/create-react-app/issues/11756#issuecomment-1001130053

**Comment by aaronmboyd** [comment]
Will there be an official fix for this issue on @celo/contractkit that works out of the box with CRA5+?

**Reply by aaronmgdr** [reply]
@aaronmboyd  yes! 

I just released 2.3.0-beta.2 and was able to turn off the fallback config so going forward it should work out of the box with CRA 5.
